commit 98cb80c5fba9550451f6df711a2ae460608acd5e
Author: Christoph Moench-Tegeder <cmt@burggraben.net>

    Bug 1973105 - remove out-of-function semicolon, r=rrelyea
    
    given the right flags, consumers of nss may fail with
    : /usr/local/include/nss/pkcs11n.h:633:92: error: extra ';' outside of a function [-Werror,-Wextra-semi]
    
    Differential Revision: https://phabricator.services.mozilla.com/D254438

diff --git lib/util/pkcs11n.h lib/util/pkcs11n.h
index cba3f63ae..9365d697a 100644
--- lib/util/pkcs11n.h
+++ lib/util/pkcs11n.h
@@ -630,7 +630,7 @@ _NSS_DEPRECATE_DEFINE_TYPE(CK_TRUST, CKT_NSS_VALID,
 #define CKT_NSS_VALID \
     _NSS_DEPRECATE_DEFINE_VALUE(CKT_NSS_VALID, CKT_NSS_NOT_TRUSTED)
 _NSS_DEPRECATE_DEFINE_TYPE(CK_TRUST, CKT_NSS_MUST_VERIFY,
-                           "CKT_NSS_MUST_VERIFY really functions as CKT_NSS_TRUST_UNKNOWN");
+                           "CKT_NSS_MUST_VERIFY really functions as CKT_NSS_TRUST_UNKNOWN")
 #define CKT_NSS_MUST_VERIFY \
     _NSS_DEPRECATE_DEFINE_VALUE(CKT_NSS_MUST_VERIFY, CKT_NSS_TRUST_UNKNOWN)
 
